sdk/tests/tutorials/ibor/test_bitemporal.py

```diff
@@ -34,16 +32,6 @@ def setUpClass(cls):
     def tearDownClass(cls):
         delete_entities(cls.id_generator)
 
-    def print_transactions(self, as_at, transactions):
-        print("transactions at: {}".format(as_at))
-
-        for transaction in transactions:
-            print("{}\t{}\t{}\t{}\t{}".format(transaction.instrument_uid,
-                                              transaction.transaction_date,
-                                              transaction.units,
-                                              transaction.transaction_price.price,
-                                              transaction.total_consideration.amount))
-
```

Comment:
Is this just because you didn't like the fact that we are somewhat randomly printing these?

Reply:
You missed my codecraft. :)   Prints and log output in tests is a really bad idea.  There's the potential to leak sensitive information to a medium that isn't sensitive to it, and if you aren't careful, something you are using for debug purposes is now spamming 1000's of lines to stdout/stderr, which can significantly hinder the task of identifying what is wrong overall.

Reply:
Yeah of course, makes sense to me, thanks for doing this.